packages/terra-application-links/src/ApplicationLinks.js

```diff
@@ -1,20 +1,29 @@
 import PropTypes from 'prop-types';
 import ApplicationTabs from './tabs/ApplicationTabs';
 
-const ApplicationLinksPropType = PropTypes.arrayOf(PropTypes.shape({
```

Comment:
@mjhenkes @bjankord @JakeLaCombe @ryanthemanuel 
Does this proptype change make this a major version bump for terra-application-links?

Reply:
Yep, this will require a major version bump. But it wont' be a big deal because we just released this a week ago and it's very unlikely to have been consumed between then and now.
Good catch calling this out.